databricks_blueprints/download_file_from_dbfs.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import os
+import sys
+import requests
+import shipyard_utils as shipyard
+try:
+    import helpers
+    import errors
+except BaseException:
+    from . import helpers
+    from . import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--instance-id', dest='instance_id', required=True)
+    parser.add_argument('--source-file-name', dest='source_file_name', required=True)
+    parser.add_argument('--source-folder-name', dest='source_folder_name', required=False)
+    parser.add_argument('--dest-file-name', dest='dest_file_name', required=False)
+    parser.add_argument('--dest-folder-name', dest='dest_folder_name', required=False)
+    args = parser.parse_args()
+    return args
+
+
+def download_file_from_dbfs(client, source_file_path, dest_file_path):
+    read_handle_endpoint = "/dbfs/read"
+    with open(dest_file_path, "w+") as file:
+        offset = 0
+        while True:
+            payload = {
+                "path": source_file_path,
+                "offset": offset, 
+                "length": 1024 # 1MB of data 
+            }
+            read_response = client.get(read_handle_endpoint, params=payload)
+            if read_response.status_codes == requests.codes.ok:
+                response_data = read_response.json()
+                data = response_data['data']
+                file.write(data)
+                # check if there is still more data left to read
+                if response_data['bytes_read'] == 1024:
+                    offset += 1024
+                else: # that was the last of the data in the file
+                    break
+            else:
+                sys.exit(errors.EXIT_CODE_DBFS_READ_ERROR)
+    print(f"finished downloading file:{source_file_path} as {dest_file_path}")
+    
+
+def main():
+    args = get_args()
+    access_token = args.access_token
+    instance_id = args.instance_id
+    source_file_name = args.source_file_name
+    source_folder_name = args.source_folder_name
+    dest_file_name = args.dest_file_name
+    dest_folder_name = args.dest_folder_name
+    # create client
+    client = helpers.DatabricksClient(access_token, instance_id)
+    # create file paths
+    if not source_folder_name:
```

Comment:
Needs to be cleaned with shipyard_utils so the placement of `/` doesn't impact anything.